scripts/generate_entity_tests.py

```
@@ -0,0 +1,249 @@
+
+        # 4. 変更があるかチェック
+        if has_git_changes() or has_unpushed_commits(base_branch):
+            # 変更がある場合: ベースブランチを更新（次のPR用）
+            print(f"\n変更あり: {branch_name} を次のベースブランチとして使用")
+            base_branch = branch_name
+        else:
+            # 変更がない場合: ブランチを破棄して元のブランチに戻す
+            print(f"\n変更なし: {branch_name} を破棄して {base_branch} に戻ります")
+            run_command(["git", "checkout", base_branch])
+            run_command(["git", "branch", "-D", branch_name])
+            # base_branchは更新しない（次のイテレーションでも同じベースを使う）
```

Ruff (0.14.10)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)